iquip/apps/builder.py

```diff
@@ -381,6 +383,11 @@ def get_procdesc(self, argInfo: Dict[str, Any]) -> Dict[str, Any]:
         }
         return procdesc
 
+    def scannable_info(self) -> Dict[str, Any]:
+        """Gets a ditctionary of scannable arguments from _ScanEntry."""
```

Comment:
"dictionary"

Reply:
In addition, I think you can make this class inherrited from `_BaseEntry` and rename `scannable_info()` to `value()`.

Of course, as you said, the layout type is different, but it might be resolved easily through modifying `_BaseEntry`.

Reply:
I missed wrong spelling....

If you want to use `BaseEntry`, its layout should change from `QHBoxLayout` to `QGridLayout`. Of course GridLayout has more option and it is possible. But one should change all the layout of basic Entries. Do you want that or not?

Reply:
Umm.. I came up with an idea! How about not modifying `_BaseEntry` and put `QGridLayout` into `self.layout`?
The example is as below.

```python
class _ScanEntry(_BaseEntry):

  def __init__(...):
    ...
    layout = QGridLayout(self)
    ...
    self.layout.addLayout(layout)
```

As I understand, you seem to want to leave the same column as the name label empty, so I think it will be the same.